src/views/mfa-verify/PushForm.js

```diff
@@ -58,12 +59,13 @@ define(['okta', 'util/CookieUtil'], function (Okta, CookieUtil) {
         deviceName: this.model.get('deviceName')
       });
     },
-    setSubmitState: function (ableToSubmit) {
+    setSubmitState: function (ableToSubmit, useRetryLabel) {
```

Comment:
So, in general, I'm wondering if we can simplify this by doing it at a lower level. Our API now times out at 5 minutes, but we've decided that we should do a client side timeout at 30 seconds. Why not just update the poll code in Factor.js (the place where you added the setTransaction call) to set a timeout response if we've hit 5 of the PUSH_INTERVALs?

If we do it this way, we don't actually need to touch the higher level code (unless you want to change the UI i.e. change the button text, but at that point it should behave the same whether you get the timeout from the server or our client side limit).